test/cosmoz-bottom-bar.html

```diff
@@ -276,7 +275,7 @@
 					bottomBar.active = true;
 				});
 
-				test('menu should be visible on screen', function () {
+				test('menu should be visible on screen', function (done) {
```

Comment:
I will update it but there will be lint errors because of it.
I will add it as global temporary.